phnt/include/ntpsapi.h

```diff
@@ -2698,6 +2697,7 @@ typedef struct _NTPSS_MEMORY_BULK_INFORMATION
     PVOID NextValidAddress;
 } NTPSS_MEMORY_BULK_INFORMATION, *PNTPSS_MEMORY_BULK_INFORMATION;
 
+#if (PHNT_VERSION >= PHNT_20H1)
```

Comment:
Because ZwPssCaptureVaSpaceBulk is also using this structure.

Reply:
But I didn't see a version checking macro in the zwapi header file.

Reply:
`Nt` vs `Zw` is meaningless for user mode. They're effectively equivalent exports. This API is not exported in the kernel, where `Zw` would be relevant. `ntzwapi.h` is just a generated header and doesn't have the guards for versions like the `Nt` versions do. Maybe it should but it's outside the scope of these changes. Regardless, user mode should almost always be using the `Nt` versions.

Reply:
![image](https://github.com/winsiderss/systeminformer/assets/32032670/275dc42e-f44d-491a-b6d3-11be566a795e)
I am curious about why ntzwapi.h is included in UserMode.

Reply:
PHNT is distributed as a standalone SDK via a separate repository (https://github.com/winsiderss/phnt) and since ZW functions are exported from ntdll.dll in usermode the header is also available - some third party projects import the SDK and call Zw functions for whatever reason but as maintainers of the SDK we don't dictate which version - We point to the documentation: https://learn.microsoft.com/en-us/windows-hardware/drivers/kernel/using-nt-and-zw-versions-of-the-native-system-services-routines

> ZwPssCaptureVaSpaceBulk is also using this structure.

ZwPssCaptureVaSpaceBulk is not exported from the kernel and this case the Zw function only exists as an artifact of autogenerating the header.

Reply:
If the PHNT_VERSION is lowered to below PHNT_20H1, phnt will become unusable. Is there a better solution available?😀